### ISSUE TITLE
损失函数

### ISSUE THREAD

**Issue by Adila9977** [slrl123/BSAFusion#10]
您好， 在模型训练时 loss = cls_loss + transfer_loss + flow_loss + fusion_loss + reg_loss， 这些损失函数都会被最小化么？

**Comment by slrl123** [comment]
是的
